src/EVM/Types.hs

```diff
@@ -15,7 +17,6 @@ import Control.Monad (mzero)
 import Control.Monad.ST (ST)
 import Control.Monad.State.Strict (StateT)
 import Crypto.Hash (hash, Keccak_256, Digest)
-import Data.Aeson
```

Comment:
This was interfering with some things through clashing with the namespace. I think it makes sense to use the `JSON.(whatever)` syntax instead, it's more explicit and it's in not too many places